st2reactor/bin/st2garbagecollector

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python2.7
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#
+#   st2 garbage_collector
+#
+
+import os
+import sys
+
+from st2reactor.cmd import garbagecollector
+
+DEVEL = True
+
+if DEVEL:
```

Comment:
That's what I asked on Slack the other days.

It seems like an old code, we have it in other service. Should verify if it's still needed.